citi-onboarding/DADe-onepage
# feat(feature/create-members-banner-and-navbar-backend-integration): create backend integration

## What I did

- Create backend integration to OurMembers, Banners and Navbar views.
- Remove some images from Assets folder that are now in the backend
- Add link input to the Banner model, to make the Participate Button (present Banner and Navbar) redirect to any link of their choice.

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/create-members-banner-and-navbar-backend-integration`
   4. `git pull origin feature/create-members-banner-and-navbar-backend-integration`
   5. `npm install`
   6. `cd server/`
   7. `node index.js`
   8. Open a new terminal
   9. `cd client/`
   10. `npm start`

2. In your browser go to http://localhost:3000  (should open automatically when running npm start)

3. Check if the all the Banner, Navbar and OurMembers views looks just like the mockup in different screen sizes (minimum width: 1366px) and if when you increase it past 1920px it remains the same size (Doesnt grow along with the screen).

4. In your browser, go to http://localhost:3001/admin/membros, create a new member or change any property of a existing one, save the changes, refresh the http://localhost:3000 and see if your changes are displayed correctly. 

5. In your browser, go to http://localhost:3001/admin/banners, change any property of a existing banner, save the changes, refresh the http://localhost:3000 and see if your changes are displayed correctly. After your changes, make sure to return the list items to its original state (text and image).

6. In your browser, go to http://localhost:3001/admin/logos, change the image property of the existing one, save the changes, refresh the http://localhost:3000 and see if your changes are displayed correctly. After your changes, make sure to return the image its original state.

8. Read the changed files section on GitHub 

9. Test the things that you think are worth testing, even the ones that are not in this description
